scripts/check-license/.gitignore

```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
```

Comment:
I see the point of adding the .gitignore in that folder and I think its sufficient for almost any case. However, maybe someone decides to initialize a pyenv within the root to cover anything from the root which will create a `.python-version` file in the root which is tracked by git. I think it would be better if we include either those rules in the repositories root `.gitignore`.

Reply:
28e8f705fe5aed271ed2c49274b01cfae896d8e6